source/developer-guides/ias.rst

```diff
@@ -1,24 +1,24 @@
 IAS Image
 -----------

```

Comment:
Should we describe in this section what "IAS" stands for?

Reply:
good catch... In earlier review, we chose to avoid disclosing its full name as they are more than one we cannot confirm.